File: codificacao-c#/URI1046.cs
using System;

class URI
{

    static void Main(string[] args)
    {
        string[] linha = Console.ReadLine().Split(' ');
        int hora1 = Convert.ToInt32(linha[0]);
        int hora2 = Convert.ToInt32(linha[1]);
        int duracao = hora1 < hora2 ? hora2 - hora1 : hora2 - hora1 + 24;
        Console.WriteLine("O JOGO DUROU " + duracao + " HORA(S)");
    }

}

File: codificacao-java/Uri1046.java
import java.io.IOException;
import java.util.*;

public class Main {

	public static void main(String[] args) throws IOException {
        Scanner sc = new Scanner(System.in);
        int hora1 = sc.nextInt();
        int hora2 = sc.nextInt();
        int duracao = hora1 < hora2 ? hora2 - hora1 : hora2 - hora1 + 24;
        System.out.println("O JOGO DUROU " + duracao + " HORA(S)");
	}
}